charts/logging-operator/templates/psp.yaml

```diff
@@ -5,8 +5,8 @@ metadata:
   name: psp.logging-operator
   namespace: {{ include "logging-operator.namespace" . }}
   annotations:
-    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default,runtime/default'
     seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default,runtime/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'runtime/default'
```

Comment:
Would it make sense to make these two configurable?

Reply:
Yes, I think so. But these changes meet our needs for now :)

Reply:
So we don't forget: https://github.com/banzaicloud/logging-operator/issues/688